packages/lib/survey/service.ts

```diff
@@ -589,7 +691,11 @@ export const duplicateSurvey = async (environmentId: string, surveyId: string) =
   return newSurvey;
 };
 
-export const getSyncSurveys = async (environmentId: string, person: TPerson): Promise<TSurvey[]> => {
+export const getSyncSurveys = async (
+  environmentId: string,
+  person: TPerson,
```

Comment:
I think we should work with the `personId` here. It just fits the rest of the services better and since it's cached it doesn't add any performance issues.
Please also validate the personId then :-)

Reply:
@pandeymangg Why was this resolved, without a comment or changes? ;-) 

I still prefer it to be `personId` instead of `person`.